recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
-{% set python_min = '3.10' %}
+{% set python_min = "3.10" %}
 {% set name = "scvi-tools" %}
-{% set version = "1.2.2.post1" %}
+{% set version = "1.2.2.post2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
-  sha256: 957bb62014212595349c934b4fc4bf3e86b9f649e4db8490f0e62901b120ee00
+  sha256: bd594fc1471c0343b34377b3ccd8603f2b640d6546432ec9027b06d36cee45bc
 
 build:
   noarch: python
-  number: 1
+  number: 2
```

Comment:
Not a huge deal, but just FYI you want to reset the build number to 0 whenever the version is updated. By bumping it to 2, the first binary uploaded for 1.2.2.post2 has build number 2 instead of the typical 0

https://anaconda.org/conda-forge/scvi-tools/files?version=1.2.2.post2

![image](https://github.com/user-attachments/assets/900cc0b2-cb86-41af-b548-f29465d27585)

No need to update anything, just something to be aware of for future update PRs. I appreciate that this update PR was complicated by my PR #67